### ISSUE TITLE
A little question about the results of the experiment

### ISSUE THREAD

**Issue by ShenAoChen2001** [hananshafi/vits-for-small-scale-datasets#6]
I trained the original vit from start to finish on the cifar100 dataset. the configuration is Vision Transformer(12 Layers,Patch-size-4,Image Size-32x32) and the accuracy is 40.4 which is consistent with the his([https://github.com/ra1ph2/Vision-Transformer]). 
I wonder how you get your data(ViT scratch 73.81 )???

**Comment by hananshafi** [comment]
You should use the finetuning recipe from our code to get 73.81 accuracy. We use different augmentations, learning rate schedular, epochs etc,
